packages/components/src/internal/components/notifications/actions.ts

```diff
@@ -44,10 +44,143 @@ export function createNotification(creatable: NotificationCreatable) {
  * Used to notify the server that the trial banner has been dismissed
  */
 export function setTrialBannerDismissSessionKey(): Promise<any> {
-    return new Promise((resolve, reject) => {
+    return new Promise(() => {
         Ajax.request({
             url: buildURL('core', 'dismissWarnings.api'),
             method: 'POST',
         });
     });
 }
+
+export function getServerNotifications(typeLabels?: string[], maxRows?: number): Promise<ServerActivity> {
+    return new Promise((resolve, reject) => {
+        Ajax.request({
+            url: ActionURL.buildURL('notification', 'getUserNotifications.api'),
+            method: 'GET',
+            params: { container: getServerContext().container.id, typeLabels, maxRows },
+            success: Utils.getCallbackWrapper(response => {
+                if (response.success) {
+                    const notifications = [];
+                    response.notifications.forEach(notification => {
+                        notifications.push(new ServerActivityData(notification));
+                    });
+                    resolve({
+                        data: notifications,
+                        totalRows: response.totalRows,
+                        unreadCount: response.unreadCount,
+                        inProgressCount: 0,
+                    });
+                } else {
+                    console.error(response);
+                    reject('There was a problem retrieving your notification data.');
+                }
+            }),
+            failure: Utils.getCallbackWrapper(response => {
+                console.error(response);
+                reject(resolveErrorMessage(response));
+            }),
+        });
+    });
+}
+
+export function getPipelineJobStatuses(): Promise<ServerActivity> {
+    return new Promise((resolve, reject) => {
+        selectRows({
+            schemaName: 'pipeline',
+            queryName: 'job',
+            filterArray: [Filter.create('Status', 'RUNNING')],
```

Comment:
No, I think we want to show it regardless of who started it since it's informative for everyone to know that a background job is running for a sample type of interest.